fairlearn/adversarial/_adversarial_mitigation.py

```diff
@@ -540,7 +541,7 @@ def fit(self, X, y, *, sensitive_features=None):
 
         return self
 
-    def partial_fit(self, X, y, *, sensitive_features=None):
+    def partial_fit(self, X, y, *, sensitive_features=None, classes=None):
```

Comment:
where is this then used?

Reply:
The estimator checks fail if it isn't there, and I found it the same way in scikit-learn's code. I assumed it must be used somewhere, I am not very familiar with `partial fit `and it use cases 😬

Reply:
scikit-learn example: https://github.com/scikit-learn/scikit-learn/blob/329a1cf4bd46664fbf81806cde82a27f44b760e0/sklearn/multiclass.py#L394C32-L394C33

Reply:
Yes I know they fail, but it doesn't make sense to take the argument and do nothing with it.

We should either have the failing tests in the xfail list, or actually fix the problem by using this argument.

Making the test pass w/o fixing the issue makes it harder to fix the problem later.

Reply:
Oh I see what you mean, it is better to xfail it until we know if there is an actual use case for it for fairlearn. Makes sense, I'll push a commit now.

Reply:
Yes, but you still have this unused argument here. Please remove the argument, to make sure later the relevant test fails.